examples/example.py

```diff
@@ -70,11 +70,11 @@ def main():
     # Settings for training model using TrainInput.
     train_input = TrainInput(
         data=train_data, target='Survived',
-        algos=[Alg.XGBoost], max_model=2, tolerance=0.9)
+        algos=[Alg.XGBoost.value], max_model=2, tolerance=0.9)
```

Comment:
.value should only appear in API request

Reply:
done